pom.xml

```diff
@@ -158,6 +158,12 @@
     <build>
         <pluginManagement>
             <plugins>
+                <plugin>
+                    <groupId>org.apache.felix</groupId>
+                    <artifactId>maven-bundle-plugin</artifactId>
+                    <version>3.5.1</version>    <!-- The last version supporting Java 7. -->
```

Comment:
Trivial: would you mind putting the comment on the preceding line?

```suggestion
                    <!-- The last version supporting Java 7. -->
                    <version>3.5.1</version>    
```

Reply:
Sure, repushed!